### DISCUSSION TITLE
YOLO v2 to YOLO v8

### DISCUSSION THREAD

**Discussion by 7thstorm** [ultralytics/ultralytics#16082] [Q&A]
Greetings,
Some years ago, I had spent considerable resources training a YOLOv2 model with about a million images (3 classes). I want to see if I can leverage the last weights file to train on v8

**Comment by glenn-jocher** [comment]
@7thstorm hello, you can transfer the knowledge from your YOLOv2 model to YOLOv8 by using the YOLOv2 weights as a starting point for training YOLOv8. This process is known as transfer learning. You may need to convert the weights to a compatible format first.

**Reply by 7thstorm** [reply]
Thanks That's good to know it is possible. Any idea on where to find a method for converting the weights from yolov2 to v8?

**Reply by glenn-jocher** [reply]
@7thstorm you can convert YOLOv2 weights to a format compatible with YOLOv8 using a framework like PyTorch. Look for scripts or tools online that facilitate this conversion process.
